apps/prairielearn/src/sync/course-db.ts

```diff
@@ -1472,8 +1477,54 @@ function validateCourseInstance({
     warnings.push('"selfEnrollment.useEnrollmentCode" is not configurable yet.');
   }
 
-  if (courseInstance.selfEnrollment.restrictToInstitution !== true) {
-    warnings.push('"selfEnrollment.restrictToInstitution" is not configurable yet.');
+  const usingLegacyAllowAccess = courseInstance.allowAccess.length > 0;
+  const usingModernPublishing = courseInstance.publishing != null;
+
+  // TODO: Remove this once the UI is merged
+  if (usingModernPublishing) {
+    warnings.push('"publishing" is not configurable yet.');
```

Comment:
Should this be a warning or an error? If you do in fact want this to be a warning, I'm almost certain that we need to sync an appropriate value for `course_instances.modern_publishing`.